### PR TITLE
added styling and class names to fit scss-lint

### PR DESCRIPTION
**Pull request by davecarter**
## Included in this PR:

- Class re-naming to fit scss-linting rules.
- Refactored Classes from realestate-favourites-list pagination component.

pagination

Please review

### CODE REVIEW

File: src/sui-pagination-item/index.jsx [hunk 1]
```diff
@@ -16,7 +16,8 @@ export default class PaginationItem extends React.Component{
       <a
         href='#'
         onClick={this.props.handleItemSelected}
-        data-pagenumber={this.props.pageNumber}>
+        data-pagenumber={this.props.pageNumber}
```
Comment (carlosvillu) [inline src/sui-pagination-item/index.jsx]: that is really necessary?

Comment (danderu) [inline src/sui-pagination-item/index.jsx]: what do you mean?

File: src/sui-pagination/_sui-pagination.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    border-right: 8px solid $c-bg-dark;
+    border-top: 6px solid transparent;
+    content: '';
+    height: 0;
+    position: absolute;
+    right: 5px;
+    top: 32%;
+    width: 0;
+  }
+}
```
Comment (ruben-martin-lozano) [inline src/sui-pagination/_sui-pagination.scss]: Hi! :)

Both .sui-Pagination-prev and .sui-Pagination-next :after styles are the same, I think that would be nice to have it just once defined, something like:

```
.sui-Pagination-next {
	padding-right: 20px;
}
.sui-Pagination-prev {
	padding-left: 20px;
}

.sui-Pagination-prev:after, .sui-Pagination-next:after {
	border-bottom: 6px solid transparent;
	border-left: 8px solid $c-bg-dark;
	border-top: 6px solid transparent;
	content: '';
	height: 0;
	position: absolute;
	right: 5px;
	top: 32%;
	width: 0;
}
```

So we can maintain it easily :)

**Comment by ruben-martin-lozano** [comment]
:+1:

**Pull request closed without merging**